website/server/models/user/schema.js

```diff
@@ -24,7 +24,21 @@ let schema = new Schema({
     local: {
       email: {
         type: String,
-        validate: [validator.isEmail, shared.i18n.t('invalidEmail')],
+        validate: [{
+          validator: validator.isEmail,
+          msg: shared.i18n.t('invalidEmail'),
+        }, {
+          validator: function notEmailDomains (email) {
+            let domains = ['habitica\\.com', 'habitrpg\\.com'];
```

Comment:
Okay, I just didn't want to exclude an email like `habitica.com@example.com` (even though that is a really contrived example)